lib/Pod/To/HTML.pm6

```diff
@@ -68,7 +69,17 @@ class Pod::To::HTML {
         my ($template-file, %partials) = retrieve-templates($!template, $!main-template-path);
         my $content = $template-file.IO.slurp;
 
-        Template::Mustache.new.render($content, %context, :from[%partials], :literal);
+        my $counter;
+        loop {
+            die $! if $counter++ == 5;
```

Comment:
So you're testing precisely five times if this works, right? Don't like magic numbers too much. Maybe clarify assigning it to a constant with a significant name?

Reply:
Also, we might want to raise an exception instead of dying. Not now, but probably later when this is accepted.

Reply:
But `die` raises an exception, just of type X::AdHoc, no? AFAIK, `die $!` just re-throws the $!, but modified it to be maybe more explicit `.throw` call.

Reply:
Correct, but I meant a class or module specific exception. It's OK, however.